Assets/Scenes/Chris' scenes/Chris Basic Test Scene.unity

```diff
@@ -879,36 +879,33 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   inventorySlots: 0
---- !u!114 &1895852632
+--- !u!114 &1895852630
 MonoBehaviour:
   m_ObjectHideFlags: 0
   m_PrefabParentObject: {fileID: 0}
   m_PrefabInternal: {fileID: 0}
   m_GameObject: {fileID: 1895852628}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: 8ce542e197bcc9945b8a93b593f886ce, type: 3}
+  m_Script: {fileID: 11500000, guid: 50837f76adcc1874e8bd9ce19f53ce03, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  currentExperience: 0
-  toNextLevel: 0
-  charLevel: 0
---- !u!114 &1895852633
+  maxHealth: 100
```

Comment:
see above comment about inspector only values. One conflict and they are all gone, not to mention when we start building a main game scene these will be lost